app/code/Magento/DownloadableGraphQl/etc/schema.graphqls

```diff
@@ -5,6 +5,35 @@ type Query {
     customerDownloadableProducts: CustomerDownloadableProducts @resolver(class: "Magento\\DownloadableGraphQl\\Model\\Resolver\\CustomerDownloadableProducts") @doc(description: "The query returns the contents of a customer's downloadable products") @cache(cacheable: false)
 }
 
+type Mutation {
+    addDownloadableProductsToCart(input: AddDownloadableProductsToCartInput): AddDownloadableProductsToCartOutput @resolver(class: "Magento\\QuoteGraphQl\\Model\\Resolver\\AddSimpleProductsToCart")
+}
+
+input AddDownloadableProductsToCartInput {
+    cart_id: String!
+    cart_items: [DownloadableProductCartItemInput!]!
+}
+
+input DownloadableProductCartItemInput {
+    data: CartItemInput!
+    downloadable_product_links: [DownloadableProductLinksInput!]
+    customizable_options:[CustomizableOptionInput!]
+}
+
+input DownloadableProductLinksInput {
+    link_id: Int!
+}
+
+type AddDownloadableProductsToCartOutput {
+    cart: Cart!
+}
+
+type DownloadableCartItem implements CartItemInterface @doc(description: "Downloadable Cart Item") {
+    customizable_options: [SelectedCustomizableOption] @resolver(class: "Magento\\QuoteGraphQl\\Model\\Resolver\\CustomizableOptions")
+    downloadable_product_links: [DownloadableProductLinks] @resolver(class: "Magento\\QuoteGraphQl\\Model\\Resolver\\DownloadableLinks") @doc(description: "An array containing information about the links for the added to cart downloadable product")
```

Comment:
Please, split resolver
One for `downloadable_product_links` and separate `downloadable_product_samples`
Thanks

Reply:
**done**